.changeset/silly-jars-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-charts/core': minor
```

Comment:
I guess this is technically major since it's not backwards compatible, but since it's still in v0, I made it minor. Let me know if this is incorrect, though!

Reply:
Yeah, pre-v1 changesets can be a bit loose. Rule of thumb for me is everything shifts down a level